### ISSUE TITLE
Do latitude/longitude range checking

### ISSUE THREAD

**Issue by webteckie** [keystonejs/keystone#3669]
### Steps to reproduce the behavior
Use an out of range value.

### Expected behavior
A nice to the point message alerts the user of the issue.

### Actual behavior
A nasty looking mongoose error is displayed.

**Comment by bassjacob** [comment]
@webteckie I'm not sure I understand the issue. Is this if someone inputs a range outside of the allowed range of lat/long?

**Comment by jstockwin** [comment]
@bassjacob Yeah, this is an issue we had when setting up tests once. We were using a lat/lon of just "1234" for each, which is not a real lat/lon and so the tests failed. It took us a while to work out why, and seemed worth implementing some kind of validation message.

**Comment by bassjacob** [comment]
yep, it looks like the field should validate that at the form level. I think this might be a larger issue where we could be using something like [this awesome ruby lib](https://github.com/trailblazer/reform) to validate fields?

**Comment by bassjacob** [comment]
I'd be happy to spike adding something like that to the field definition at a keystone level to give more granular allowed inputs?

**Comment by jstockwin** [comment]
Fine by me. As I said, it was just something that we noticed and thought we'd flag up :)
